src/styles/_catalog.scss

```
@@ -14,6 +14,11 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all ease 300ms;
```

```suggestion
    transition: all ease 300ms;
```
Не використовуй transition all, опиши кожну властивість яка має змінюватись